doc/install/python.rst

```diff
@@ -66,77 +71,74 @@ Every time you make changes to the code, you'll have to repeat the second step.
-below demonstrate this for both pip and ``setup.py``.
+* ``--vec``: Enable vectorization. This requires choosing an appropriate architecture using ``--arch``.
+  Note that on x86-64 platforms compilation will fail if you enable vectorization, but the CPU or ``--arch`` does not support any form of AVX.
+* ``--arch``: CPU micro-architecture to target. The advised default is ``native``.
+  See `here <https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html>`_ for a full list of options.
 
 **Vanilla install** with no additional features enabled:
 
 .. code-block:: bash
 
     pip3 install arbor
-    python3 ./arbor/setup.py install
 
 **With MPI support**. This might require loading an MPI module or setting the ``CC`` and ``CXX``
 :ref:`environment variables <install-mpi>`:
 
 .. code-block:: bash
 
     pip3 install --install-option='--mpi' ./arbor
-    python3 ./arbor/setup.py install --mpi
 
 **Compile with** :ref:`vectorization <install-vectorize>` on a system with a SkyLake
 :ref:`architecture <install-architecture>`:
 
 .. code-block:: bash
 
     pip3 install --install-option='--vec' --install-option='--arch=skylake' arbor
-    python3 ./arbor/setup.py install --vec --arch=skylake
 
 **Enable NVIDIA GPUs (compiled with nvcc)**. This requires the :ref:`CUDA toolkit <install-gpu>`:
 
 .. code-block:: bash
 
     pip3 install --install-option='--gpu=cuda' ./arbor
-    python3 ./arbor/setup.py install  --gpu=cuda
 
 **Enable NVIDIA GPUs (compiled with clang)**. This also requires the :ref:`CUDA toolkit <install-gpu>`:
 
 .. code-block:: bash
 
     pip3 install --install-option='--gpu=cuda-clang' ./arbor
-    python3 ./arbor/setup.py install --gpu=cuda-clang
 
 **Enable AMD GPUs (compiled with hipcc)**. This requires setting the ``CC`` and ``CXX``
 :ref:`environment variables <install-gpu>`
 
 .. code-block:: bash
 
     pip3 install --install-option='--gpu=hip' ./arbor
-    python3 ./arbor/setup.py install --gpu=hip
 
 .. Note::
-    Setuptools compiles the Arbor C++ library and
-    wrapper, which can take a few minutes. Pass the ``--verbose`` flag to pip
+    Setuptools compiles the Arbor C++ library and wrapper, as well as dependencies you did not have installed
+    yet (e.g. `numpy`). It may take a few minutes. Pass the ``--verbose`` flag to pip
     to see the individual steps being performed if you are concerned that progress
     is halting.
 
+    If you had Arbor installed already, you may need to remove it first before you can (re)compile
+    it with the flags you need.
+
 .. Note::
     Detailed instructions on how to install using CMake are in the
     :ref:`Python configuration <install-python>` section of the
     :ref:`installation guide <in_build_install>`.
-    CMake is recommended for developers, integration with package managers such as
-    Spack and EasyBuild, and fine-grained control over compilation and installation.
+    CMake is recommended if you need more control over Arbors compilation, plan to use Arbor with C++,
```

Comment:
`control over Arbors compilation` -> `control over compilation and/or installation`